change/@fluentui-react-positioning-d46e41bd-237b-4c6e-bb3f-b3d88a836fdd.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
suggestion for future: IMO this should have been type `none` , no release needs to be triggered based on these changes